superdesk/default_settings.py

```diff
@@ -968,3 +968,4 @@ def local_to_utc_hour(hour):
 #: .. versionadded:: 2.1
 #:
 APPS_DATA_UPDATES_PATHS = []
+PUBLISH_ASSOCIATIONS_RESEND = "default"
```

Comment:
can you call the default value `new` as `default` doesn't say much about the behaviour and document in the comment above the other options?